src/scrapper/database/dbs/create_db_sqlite3.sql

```diff
@@ -146,3 +146,26 @@ CREATE TABLE `info` (
 );
 
 -- --------------------------------------------------------
+--
+-- Table for `course group`
+--
+CREATE TABLE `course_group` (
+  `id` INTEGER PRIMARY KEY,
+  `name` varchar(64),
+  `course_id` INTEGER,
+  `group_course_id` INTEGER,
```

Comment:
Why is this needed?

Reply:
Basically the ID is NOT an hash of the course_id and group_course so basically before inserting a new element I need to check if it already exists using those two. If it was an hash they could be removed.